ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -8237,3 +8237,49 @@ def test_torch_instance_multiply(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# TODO: need to fix this test after fixing the tests reults from torch.unique
+# unique
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="unique",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        force_int_axis=True,
+        valid_axis=True,
+        min_num_dims=1,
```

Comment:
We shouldn't limit the dimension size, `torch.unique` works fine with 0-dim tensors.